infra/modules/service/variables.tf

```diff
@@ -281,6 +281,8 @@ variable "scheduled_jobs" {
     task_command        = list(string)
     schedule_expression = string
     state               = string
+    cpu                 = optional(number, 256)
+    mem                 = optional(number, 512)
```

Comment:
Wouldn't this lower what the jobs run with currently? I think right now it's 768 (.75 cpu) and 2gb (4gb in prod). Could we set the defaults to be higher / use everything we have available?